kalite/topic_tools/__init__.py

```diff
@@ -1,618 +0,0 @@
-"""
-TODO: NOTHING SHOULD BE HERE! It's prohibiting the import of other topic_tools.xxx
```

Comment:
i18n is the only one left with code in its __init__.py, after I deleted the entire caching app. We should make sure to finish that in 0.16 too.